rpc/js/package.json

```diff
@@ -1,25 +1,27 @@
 {
   "name": "@viamrobotics/rpc",
-  "version": "0.1.22",
+  "version": "0.1.23",
   "license": "Apache-2.0",
   "dependencies": {
     "@improbable-eng/grpc-web": "^0.13.0",
-    "google-protobuf": "^3.14.0",
-    "@types/webrtc": "^0.0.31"
```

Comment:
This should already be included in Typescript's DOM library. I believe this package was meant for bleeding edge stuff that TS refused to add due to it not being browser-standard yet.